tests/context-menu.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test, expect } from '@playwright/test';
+import { waitForCustomEvent } from './lib/helper.js';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/playground/context-menu-test.html');
+});
+
+test('Renders labels, dividers, and icons', async ({ page }) => {
+  const contextMenu = page.getByTestId('context-menu-default');
```

Comment:
I think in these tests we should strive to minimize `getByTestId`. I think using it to select to overall component for the given `test` is good, but other usages are questionable